testproject/Assets/Tests/Runtime/MultiprocessRuntime/ThreeDText.cs

```diff
@@ -38,6 +40,10 @@ public void Start()
         public void Update()
         {
             m_UpdateCounter++;
+            if (MultiprocessOrchestration.IsPerformanceTest)
```

Comment:
Even better would be in Awake to SetActive false the whole gameObject, this way Start wouldn't execute either (since start is reading from a json file, this could take long enough to affect test performances?)